user.go

```diff
@@ -39,6 +43,58 @@ func (a *App) GetUser(userID string) (*User, error) {
 	}
 }
 
+// GetUserByIdentifier gets a user using their identifier
+// returns user on success, error on failure
+func (a *App) GetUserByIdentifier(identifier string) (*User, error) {
+	var errorText string
+	message := "failed to get Passage User By Identifier"
+	limit := 1
+	ilikeIdentifier := "like:(?i)" + identifier
```

Comment:
I'm sorry for the confusion -- I meant to say that your implementation of `ToLower` works since we lowercase on the backend. Although this works too, if you prefer to keep the lowercase version because it is an exact matcher vs a regex matcher I would be good with that as well.

Reply:
Ah gotcha. I am going to change it back to exact matcher since it is usually more performance than regex matcher. Thanks for the pro tip though! I did not know about the `(?i)`.